ogc-server-statistics/src/main/java/org/georchestra/ogcservstatistics/dataservices/DeleteAllCommand.java

```diff
@@ -37,23 +39,10 @@
 	 */
 	@Override
 	public void execute() throws DataCommandException {
-
-		//PreparedStatement pStmt=null;
-		Statement pStmt=null;
-        try {
-			pStmt = this.connection.createStatement();
-			pStmt.execute("DELETE FROM ogcstatistics.OGC_SERVICES_LOG");
-			
+        try (Statement pStmt = this.connection.createStatement()){
+			pStmt.execute(String.format("DELETE FROM %s", QUALIFIED_TABLE_NAME));
```

Comment:
Right. In this case it's safe as `QUALIFIED_TABLE_NAME` is a constant to avoid possible typos. It would also make it safe in the event the table name changed (e.g. it has the schema hardcoded, might that change eventually?).

Reply:
No plan to change this. 
Honestly, this looks safe enough for me.